data/categories.txt

```diff
@@ -1178,7 +1179,7 @@ pl:Sklep spożywczy
 pt:Loja de conveniência|Minimercado
 pt-BR:Loja de conveniência|4mercadinho|compras
 ro:4Magazin mixt|minimarket|magazin
-es:Tienda de barrio
+es:Tienda de barrio|chino|provisión|almacén|minimercado
```

Comment:
Yes, i know it's polemic. We cannot control how the people use the words. In most of the cities of Argentina were i have been, if you look for a shop=convenience you ask to the person near you "_¿Donde hay un chino?_" ("Where is a "chino"?) you and all other people clearly understand that you are referring to a shop and not a demonym, so why not ask to OM "chino" and get the convenience shops? An app should speak the language of their users, and not viceversa.

Besides that, it's an accepted term by the RAE. See https://dle.rae.es/chino (8. m. coloq. Bazar o supermercado regentado por chinos. Compré el cuaderno en un chino. ) 

Yes, it will interfere with a chinese food restaurant, but that is because it is another accepted meaing of this word. (RAE 7. m. coloq. Restaurante de comida china. ). Same word refers to both type of elements. There is no way to not interfere. If you decide to show one, you decide to hide the other.

The other terms "Tienda de conveniencia","Colmado","Ultramarinos" are ok for me.

¡Agradezcamos las variantes y el poder de expresion de nuestra lengua! _Let us be grateful for the variants and power of expression of our (spanish) language!_

Reply:
I support you in making the language as universal as possible, but disagree to include it and any other slang language. I know it's inconvenient as well because sometimes it's hard to find the specific official word for these places, but there are.

If you would start including slang, you could also include "guita" or "plata" for money and that IMO is not appropriate for an app. Besides, it's already important that it interferes with the results for Chinese cuisine. 

I won't strongly oppose this particular change. I'd let it be, but it would be positive if more native speakers could give their opinion.

Reply:
In english there is this one term "slang", but in spanish we also have variants. The semantic of "slang" in english, is not the same as "_coloquial_" in spanish. "slang" is more llike "_jerga_". "_Coloquial_" is more like "informal" or "casual"

I also don't want to start including slang terms (like in "_jerga_"), but useful "_coloquial_" terms used by entire populations, to make OM more useful to more people, 

I did not include "_guita_" or "_plata_" (nor most of the 20 terms with the same "cash" meaning - https://dle.rae.es/guita ) because i didn't see useful. If i need "_plata_" (money) i search for "_cajero_" (ATM). If people needing money uses "_plata_" or "_guita_" as a search term, i would consider them useful terms and ask for inclusion in the list. Just to improve usability, not for purity of language or being politically correct.

As for the interference with chinese cuisine, we should better discuss in other thread, because OM is not returning any useful results when I search for "_cocina china_" OM didn't show any of the 6 cuisine=chinese that i have < 20km from me. Same for "_cocina mexicana_" or any other type of cuisine. I opened an issue for that a few days ago (only was aware of "_cocina mexicana_" at that time.

Reply:
> I support you in making the language as universal as possible, but disagree to include it and any other slang language. I know it's inconvenient as well because sometimes it's hard to find the specific official word for these places, but there are.

The problem is not about finding the specific official word. The problem is that if any es-* user searches for "chino" in OM, and OM didn't show the shop=convenience elements around the user, the app didn't worked as expected. It's not the user which needs to know which specific term we decide to include in OM to get the results, it's OM to try to understand whatever the user is searching for, and try to solve it in the better possible way.

The issue is not about which translation of shop=convenience should OM display to the user when it shows info about a shop=convenience, the user will learn which are these "OM official" terms using the app, but what meaning gives OM to the user input. OM is not forcing or promoting the user to use "chino" or "plata", but if the user put those search terms in the text input box, I think that it is useful that OM understand what the user wants and returns the desired elements.

Reply:
@patepelo If users use slang to find objects on the map, then these keywords should provide reliable results. Imagine if Google didn't work with really used slang words.

Reply:
Ok, it's not wrong to use slang in general, is just in this case that is polemic as explained for example in [this article](https://www.elperiodico.com/es/sociedad/20230524/atencion-lenguaje-chino-paqui-educacion-75987499) or [this one](https://www.elnacional.cat/es/confidencial/ni-paki-ni-chino-campana-antirracista-revoluciona-barcelona_1149517_102.html)

But let's suppose racism sensitivity is not a problem here. We encounter a second problem that is ambiguity, with these definitions:

> 7. m. coloq. Restaurante de comida china.

 --> (coloq. = slang) A restaurant that sells chines food

> 8. m. coloq. Bazar o supermercado regentado por chinos. Compré el cuaderno en un chino

. --> (coloq. = slang) A supermarket or a variety store that's attended by Chinese people.

So if you were in Spain and you say let's search for "chino" in the app, a user here would expect to find a variety store. So now you have a search term that isn't too specific.

But as said. I'll let it be and vote abstain. It won't make of big difference.

Reply:
So then "chino" should also search for variety store. And if the user needs specific results, he always can use more specific search terms.

The polemic discussed in those articles is another issue, because I'm not proposing that OM display an POI amenity=convenience or shop=variety_store as "Chino", OM should display them as "Tienda de barrio" or "Bazar", as it does now, so OM does not promote racism or racist terms in any way.

The problem here is not about racism, nor anti-racism, nor discrimination, nor politics, it is technical, it is about how OM understand users natural language and gives the users the results they need.

BTW, it could be seen the other way, as OM is not displaying these shops in the results when the user searches for them, it can be seen as if OM is avoiding directing customers to these shops and in consequence decreasing their income.